parcels/field.py

```diff
@@ -204,10 +203,12 @@ def __init__(
         else:
             raise ValueError("Unsupported mesh type in data array attributes. Choose either: 'spherical' or 'flat'")
 
-        if allow_time_extrapolation is None:
-            self.allow_time_extrapolation = True if len(getattr(self.data, "time", [])) == 1 else False
-        else:
-            self.allow_time_extrapolation = allow_time_extrapolation
+        # Check if time is in self.data.dims
+        if "time" not in self.data.dims:
+            # Add time dimension of length 1 if not present
+            # While the choice of actual date is arbitrary, it should be a datetime object
+            # Here, we use the current time
+            self.data = self.data.expand_dims({"time": [datetime.now()]})
```

Comment:
Is this not a bit dangerous? Because it means that the result of this operation is different every time a user runs it. That can become a debugging headache. Better to set a neutral value? E.g. 1 Jan 2000? Or (as an inside-joke) 27-Sept-2015, the day of the first Parcels commit?

Reply:
Change made in https://github.com/OceanParcels/Parcels/pull/1999/commits/80eba8860e7ce7d76918068c92cde64121308ce9